src/deadline/max_adaptor/MaxAdaptor/adaptor.py

```diff
@@ -33,7 +33,33 @@ class MaxNotRunningError(Exception):
 
 # Renderer needs extra steps
 _FIRST_MAX_ACTIONS = ["scene_file", "state_set"]  # Actions which must be queued before any others
-_MAX_INIT_KEYS = {"camera", "output_file_path", "output_file_name", "output_file_format"}
+
+# Render elements feature on/off control key
+_ENABLED_MODIFY_RENDER_ELEMENTS_KEY = "enabled_modify_render_elements"
+
+# Render elements related keys
+_RENDER_ELEMENT_KEYS = {
+    "render_elements",
+    "render_elements_update_paths",
+    "render_elements_include_name_in_path",
+    "render_elements_include_type_in_path",
+    "render_elements_include_name_in_filename",
+    "render_elements_include_type_in_filename",
+    "vray_render_elements_vfb_control",
+    "vray_split_buffer_support",
+    "ignore_render_elements_by_name",
+}
+
+# Base initialization keys
+_BASE_INIT_KEYS = {
+    "camera",
+    "output_file_path",
+    "output_file_name",
+    "output_file_format",
+}
+
+# Combined initialization keys
+_MAX_INIT_KEYS = _BASE_INIT_KEYS | {_ENABLED_MODIFY_RENDER_ELEMENTS_KEY} | _RENDER_ELEMENT_KEYS
```

Comment:
I don't think we need to combine this, do we? We're combining all the render element data into a single object and passing that in one action in the new code. Having all the render element keys here means we send one action per render element key (see https://github.com/aws-deadline/deadline-cloud-for-3ds-max/blob/50520b1afa7b46897a1c0ae22e73579e8d204b62/src/deadline/max_adaptor/MaxAdaptor/adaptor.py#L291-L293)